frontend/src/components/present-poll/present-footer/present-footer.scss

```diff
@@ -0,0 +1,7 @@
+.present-footer {
+  background: black;
+
+  display: flex;
+  flex-shrink: 0;
+  overflow-x: scroll;
```

Comment:
overflow-x: auto;
to get rid of the scroll bar at the bottom of the present questions page

Reply:
THAT'S WHAT IT WAS. I thought it was a gray background and had been messing with fixing the height for so long before I gave up.